modules/openapi-generator/src/main/resources/go/model_simple.mustache

```diff
@@ -168,6 +168,18 @@ func (o *{{classname}}) Set{{name}}(v {{vendorExtensions.x-go-base-type}}) {
 {{/isNullable}}
 }
 
+{{^isReadOnly}}
+{{#defaultValue}}
+{{#initRequiredVars}}
+func (o *{{classname}}) GetDefault{{baseName}}() interface{}  {
+	var {{baseName}} {{{dataType}}} = {{{defaultValue}}}
+	o.{{name}} = {{baseName}}
+	return {{baseName}}
+}
+{{/initRequiredVars}}
+{{/defaultValue}}
+{{/isReadOnly}}
```

Comment:
Better wrap the whole block inside `{{#useDefaultValueInRequiredVars}} ... {{/useDefaultValueInRequiredVars}}` instead

Reply:
I have addressed